spec/system/applying/benefit_check/benefit_check_retry_spec.rb

```diff
@@ -54,12 +54,12 @@
       save_and_continue
 
       # steps/dwp/benefit-type
-      mock_benefit_check('No')
+      mock_benefit_check('Undetermined')
```

Comment:
Changed this to undetermined as the describe merhod above says the scenario is from undetermined to confirmed